eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -221,6 +221,17 @@ jobs:
             node eng/tools/rush-runner.js build:test --verbose -p max
           displayName: "Build test assets"
 
+        - task: NodeTool@0
+          inputs:
+            versionSpec: $(NodeTestVersion)
+          displayName: "Use NodeJS $(NodeTestVersion)"
+
+        - script: |
```

Comment:
Given some of our other recent issues around java for example perhaps instead of having the node version outputted here you create a template that will verify that the node version matches what we expect.

Reply:
Good idea, I will do this.  I agree verifying is better than just printing the version.

Reply:
I will add this in a separate PR.